app/api/rest_api.py

```diff
@@ -59,7 +59,7 @@ async def landing(self, request):
         if not access:
             return render_template('login.html', request, {})
         plugins = await self.data_svc.locate('plugins', {'access': tuple(access), **dict(enabled=True)})
-        data = dict(plugins=[p.display for p in plugins], errors=self.app_svc.errors + self._request_errors(request))
+        data = dict(plugins=[p.display for p in plugins], errors=self.app_svc.errors + self._request_errors(request), version='2.6.6')
```

Comment:
change hard-coded-ness

Reply:
well.. that hard-coded-ness will just shift location until the other PRs go in with the implementation of version grabbing

Reply:
pushed hopefully an agreeable intermediary format